### ISSUE TITLE
infra v2: migrate parking-offstreet-meranobolzano

### ISSUE THREAD

**Issue by clezag** [noi-techpark/bdp-commons#643]
This is an issue stub. Include information about the data collector here

**Comment by clezag** [comment]
Data collector has been migrated. Merano part currently cannot be tested because of (probably) IP blocking on the request when coming from the cluster.

Unclear from where the GPS location comes in production, no evidence of it in the code, but stations have a location

**Comment by clezag** [comment]
@rcavaliere 
It looks to me like parking merano (URL http://84.18.132.129/trafficstudiorestwebservice/getParkingAreasSituation) is IP blocking requests from our new infrastructure. can you confirm?

**Comment by rcavaliere** [comment]
@clezag, yes, I need to find out who can manage this now in the Municipality of Merano, so that an additional IP address can access this data.

**Comment by rcavaliere** [comment]
@clezag this was an old issue, that was however fixed. The data from the Municipality of Merano is available in real-time, check parking.bz.it

**Comment by rcavaliere** [comment]
@clezag ah yes, now I got it. Here it is likely that we are going to have a new source to be integrated, since the Municipality is changing parking back-end system. If confirmed, I would then migrate the new Data Collector in the new infrastructure, of course.
